src/polyfill.js

```diff
@@ -0,0 +1,36 @@
+export const getExternalPolyfill = (template) => template.statement.ast`
+  import { ESXToken } from "@ungap/esx";
```

Comment:
as you suggested, I've called it `@ugap/esxtoken`

Reply:
uhm ... also, it's `import ESXToken from "@ungap/esxtoken";` as I've exported a *default* ... am I a bad person? I won't mind exporting a named module filed instead

Reply:
I noticed and updated. I use default exports without problems, I don't know why I picked a named one here!